src/openrct2/Date.h

```diff
@@ -44,3 +69,28 @@ namespace OpenRCT2
         static int32_t GetDaysInMonth(int32_t month);
     };
 } // namespace OpenRCT2
+
+struct RealWorldDate
+{
+    uint8_t day;
+    uint8_t month;
+    int16_t year;
+    uint8_t day_of_week;
+};
+
+struct RealWorldTime
+{
+    uint8_t second;
+    uint8_t minute;
+    uint8_t hour;
+};
+
+extern const int16_t days_in_month[MONTH_COUNT];
```

Comment:
There is already a function to access this in Date called GetDaysInMonth, probably best to eliminate this extern and instead replace all raw access to this with Date::GetDaysInMonth